src/propslib/DelegationConfig.cpp

```diff
@@ -84,6 +84,13 @@ HRESULT _lookupCatalog(PCWSTR extensionName, std::vector<T>& vec) noexcept
         RETURN_IF_FAILED(extensionPackage2->get_PublisherDisplayName(publisher.GetAddressOf()));
         extensionMetadata.author = std::wstring{ publisher.GetRawBuffer(nullptr) };
 
+        ComPtr<IUriRuntimeClass> logoUri;
+        RETURN_IF_FAILED(extensionPackage2->get_Logo(logoUri.GetAddressOf()));
+
+        HString logo;
+        RETURN_IF_FAILED(logoUri->get_AbsoluteUri(logo.GetAddressOf()));
+        extensionMetadata.logo = std::wstring{ logo.GetRawBuffer(nullptr) };

```

Comment:
Should we `RETURN_IF_FAILED` if we failed to get the logo? Or should we `LOG_IF_FAILED` and use a placeholder instead?

Reply:
You're probably right. We're probably fine sans-logo.